wework/src/styles/globals.css

```
@@ -659,6 +659,25 @@ button:disabled {
+  }
+
+  .composer-prosemirror-editor:has(.composer-empty-caret) {
+    caret-color: transparent;
+  }
+
+  .composer-empty-caret {
+    display: inline-block;
+    width: 1px;
+    height: 1em;
+    margin-right: -1px;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)